domains/vikyaiyer.json

```diff
@@ -0,0 +1,13 @@
+{
+    "description": "Vignesh Aiyer is a Developer!",
+    "repo": "https://github.com/vikyaiyer/jekyll-theme-minimal-resume",
+    "owner": {
+      "username": "vikyaiyer",
+      "email": "vikyaiyer1997@gmail.com"
+    },
+    "record": {
+      "CNAME": "https://vikyaiyer.github.io/jekyll-theme-minimal-resume",
```

Comment:
CNAME records cannot be a full url. It has to be a host name. I'd suggest you rename your repo to `vikyaiyer.github.io`.

You can take a look at the [github pages guide](https://guides.github.com/features/pages/).

Reply:
I have updated the url

Reply:
Remove the `https://` from the cname and it'll be good to merge

Reply:
removed✌️